util/src/main/java/io/kubernetes/client/util/generic/GenericKubernetesApi.java

```diff
@@ -711,7 +711,7 @@ public GenericKubernetesApi(
       }
       V1Status status = apiClient.getJSON().deserialize(e.getResponseBody(), V1Status.class);
       if (null == status) { // the response body can be something unexpected sometimes..
-        throw new RuntimeException(e.getCause());
+        throw new RuntimeException(e);
```

Comment:
@yue9944882 i don't think this works in the case when the body cannot be deserialized to V1Status.
the `deserialize` call itself throws a `JsonSyntaxException`